webextension/cs/textarea.js

```diff
@@ -341,7 +341,16 @@ function showMatchedResultOnMarker(result) {
     Tools.getStorage().get(
     {
       ignoreQuotedLines: false,
-      ignoredRules: [],
+      ignoredRules: [
```

Comment:
Now this is duplicated? Can't we have this in a common file to avoid duplication?

Reply:
sure, we could do that.

Reply:
@danielnaber it's done. Could you take a look? Thanks.